controllers/component_build_controller_common.go

```diff
@@ -92,6 +97,33 @@ func getGitProvider(component appstudiov1alpha1.Component) (string, error) {
 	return gitProvider, err
 }
 
+// GetPipelineFromComponentAnnotation parses pipeline annotation on component and returns pipeline
+func GetPipelineFromComponentAnnotation(ctx context.Context, component *appstudiov1alpha1.Component) (*tektonapi.PipelineRef, error) {
```

Comment:
Function name nor comment says that we are talking about build pipeline.

Reply:
renamed